src/palimpzest/elements/pzlist.py

```diff
@@ -1,4 +1,4 @@
-from palimpzest import Field
+from .elements import Field
```

Comment:
I've tried to avoid local imports, so I might just push a change to make this `from palimpzest.elements import Field`